keepassxc-browser/background/page.js

```diff
@@ -187,7 +187,10 @@ page.switchTab = async function(tab) {
     browser.contextMenus.update('fill_attribute', { visible: false });
 
     // Clears all logins from other tabs after a timeout
-    clearTimeout(page.clearCredentialsTimeout);
+    if (page.clearCredentialsTimeout) {
```

Comment:
This seems like an unrelated change. Are there guarantees as to the life cycle of this object? This check implies to me that we've had trouble making sure it is valid.